pkg/infrastructure/aws/sdk/aws.go

```diff
@@ -61,7 +61,7 @@ type output struct {
 }
 
 // Provision creates cluster infrastructure using AWS SDK calls.
-func (a InfraProvider) Provision(dir string, parents asset.Parents) ([]*asset.File, error) {
+func (a InfraProvider) Provision(_ context.Context, dir string, parents asset.Parents) ([]*asset.File, error) {
```

Comment:
we could actually use the `ctx` passed in here at https://github.com/openshift/installer/pull/8063/files#diff-11e1935b1e67da8aecbfdf7885818db5002b886505d29d8aead4c19a3a29776cR112:
```go
ctx, cancel := context.WithTimeout(ctx, 30*time.Minute)
```
But probably not worth it since this is going to be replaced by capa soon.

Reply:
Will take a look when I'm fixing up lint issues